database/src/main/scala/cromwell/database/SqlDatabase.scala

```diff
@@ -222,11 +220,15 @@ trait SqlDatabase extends AutoCloseable {
     *
     * @param startMetadataId        The minimum ID an entry in `METADATA_JOURNAL` must have to be examined for summary.
     * @param startMetadataTimestamp An optional timestamp.  If specified, a metadatum must have a timestamp greater than or equal to this value.
+    * @param getUpdatedSummary      Takes in the optional existing summary and the metadata, returns the new summary.
+    *
     * @return A `Future` with the maximum ID value of the metadata summarized by the invocation of this method.
     */
-  protected def refreshMetadataSummaries(startMetadataId: Long, startMetadataTimestamp: Option[OffsetDateTime])
+  protected def refreshMetadataSummaries(startMetadataId: Long, startMetadataTimestamp: Option[Timestamp],
```

Comment:
Gotta run, so I can't summarize the links below at the moment. In short, even if a specific database can store a datetime+zone in a single column, jdbc only supports the SQL `TIMESTAMP`, not `TIMESTAMPZ`. There's some work to get slick to marshal the type, but when it's not supported by the server, the current draft is marshaling the values as strings.

- https://mariadb.com/kb/en/sql-99/08-temporal-values/
- https://en.wikipedia.org/wiki/SQL#Data_types
- https://java.net/jira/browse/JPA_SPEC-63
- https://github.com/slick/slick/issues/1026
- http://stackoverflow.com/questions/34608650/scala-slick-3-0-implicit-mapping-between-java8-offsetdatetime-and-timestamp
- https://github.com/xavier-fernandez/slick/blob/53683920c553a476b4a87c1ca296ce0c76504bc7/slick/src/main/scala/slick/jdbc/JdbcTypesComponent.scala#L310-L320